static/library/map_component.ts

```diff
@@ -199,7 +204,10 @@ export class DatacommonsMapComponent extends LitElement {
           parentPlace: placeDcid,
           variable: {
             date,
-            denom: "",
+            denom:
+              this.perCapita && this.perCapita.includes(this.variable)
```

Comment:
I think you want `this.perCapita.includes(variable)` here

Reply:
Good catch